docker-compose.yml

```diff
@@ -1,4 +1,3 @@
-version: "3.8"
```

Comment:
Do not add any changes to the docker compose file.

Reply:
['version' is obsolete](https://docs.docker.com/reference/compose-file/version-and-name/), but I can revert is, when you wish